docs/api.rst

```diff
@@ -2,4 +2,8 @@
 API reference
 =============
 
-…
+.. autofunction:: form2request.request_from_form
```

Comment:
I wonder if it'd be easier to remember if a function is called form2request. No strong opinion though.